### PR TITLE
Develop+libretexts formats

### PR DESCRIPTION
**Pull request by mgage**
CSS settings that match the use of webwork in Libretexts  (libretexts.org).

There is a potential for continual adjustments as the css for webwork and libretexts evolve so I have created a new theme libretexts (which initially was math4)  and an xmlhtml format
libretexts_format.pl which was initially sticky_format.

I would like to be able to override the css for knowles with entry in libretexts/libretexts.css,
specifically set the bottom-border to 0 width but I couldn't  make the entry in libretexts.css take precedence.  

This PR  replaces PR #1101

### CODE REVIEW

**Comment by mgage** [comment]
This pull request contains tweaks to the simple_format themes which are used when embedding problems into Libretext and other HTML pages. In particular the final lines of a problem listing the webwork and pg release numbers and copyright info are hidden inside a knowl so that they don't distract when several problems are embedded in a page

**Comment by mgage** [comment]
You can use this to test the changes:

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8" />
	<title>libretexts format</title>
	<meta name="generator" content="BBEdit 13.0" />
</head>
<body>
<iframe allowtransparency="true" 
		frameborder="0" 
		width = 80%
		height= 600
		src="https:/demo.webwork.rochester.edu/webwork2/html2xml?
		&answersSubmitted=0&
		&sourceFilePath=Library/BentleyUniversity/MathematicsOfComputerGraphics/setMoCG-Ch10/10-1-to-3-VectorValuedTrigFunctions.pg&
		&problemSeed=1234567&
		&courseID=daemon_course&
		&userID=daemon&
		&course_password=daemon&
		&showSummary=1&
		&displayMode=MathJax&
		&problemIdentifierPrefix=102&
		&language=en&
		&outputformat=libretexts"  
		></iframe>
</body>
</html>

**Comment by Alex-Jordan** [comment]
This PR mostly adds new files for the new theme, and I plan to check things out soon.

But in skimming the diff, I see two files where I wonder if there was an accidental commit. One adds the file ` htdocs/themes/math4/libretexts.template`. Is it really the case that this new template should be added in `math4`? The other is that ` htdocs/themes/math4/system.template` gets an inline styling `display:none` on its footer. Is that intentional in `math4`? And if so, is there a reason to do it this way instead of in `math4.css`?

File: lib/WeBWorK/ContentGenerator/Problem.pm [hunk 1]
```diff
@@ -2252,6 +2252,11 @@ sub output_JS{
         print qq{
            <script type="text/javascript" src="$site_url/js/vendor/underscore/underscore.js"></script>
            <script type="text/javascript" src="$site_url/js/legacy/vendor/knowl.js"></script>};
+           
+    # This is for when the problem is embedded in an iframe
+    	print qq{
+    		<script type="text/javascript" src="/webwork2_files/js/vendor/iframe-resizer/js/iframeResizer.contentWindow.min.js"></script>
+    	};
```
Comment (Alex-Jordan) [inline lib/WeBWorK/ContentGenerator/Problem.pm:2259]: Is this needed? I think of `Problem.pm` as being for rendering a problem using `webwork2`. Is there an iframe embedding technique that uses `Problem.pm` this way? In other situations like with PreTeXt, iframe-resizer is in the WebworkClient template format file.

File: htdocs/themes/math4/simple.template [hunk 2]
```diff
@@ -227,21 +227,19 @@ var tabberOptions = {manualStartup:true};
...
-
 <!-- Bootstrap Fluid Layout Scaffolding stop -->
 </div> <!-- row-fluid -->
 </div> <!-- container-fluid -->
 
 
+
 <!-- Footer -->
-<div id="footer" role="contentinfo">
+<div id="footer" role="contentinfo" style="display: none">
```
Comment (Alex-Jordan) [inline htdocs/themes/math4/simple.template:242]: Is adding `style="display: none"` intentional? And if this is needed, any reason not to use `math4.css`?

File: htdocs/themes/math4/libretexts.template [hunk 3]
```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
```
Comment (Alex-Jordan) [inline htdocs/themes/math4/libretexts.template:1]: This file seems out of place: a `libretexts.template` in the `math4` folder.

**Review by Alex-Jordan — COMMENTED** [review]
Most of this is brand new and libretext-specific, so it seems fair to just approve it and if there are bugs, presumably LibreText will deal with that or file reports. But a few things are different, and I made some localized comments.

**Comment by dlglin** [comment]
Closed due to inactivity.  Can be reopened if activity resumes.

**Pull request closed without merging**